stdlib/asyncio/tasks.pyi

```diff
@@ -270,6 +270,9 @@ def run_coroutine_threadsafe(coro: _FutureT[_T], loop: AbstractEventLoop) -> con
 
 if sys.version_info >= (3, 10):
     def shield(arg: _FutureT[_T]) -> Future[_T]: ...
+    @overload
+    async def sleep(delay: float) -> None: ...
+    @overload
     async def sleep(delay: float, result: _T = ...) -> _T: ...
```

Comment:
```suggestion
    async def sleep(delay: float, result: _T) -> _T: ...
```
So the overloads don't overlap.

Reply:
This one doesn't seem to cause errors in CI. Should I still commit this change?

Reply:
I just committed it for you :) I'm not sure why it's not causing errors in CI. I feel like it should do.